examples/hello/composer.json

```diff
@@ -1,5 +1,5 @@
 {
     "require": {
-        "google/cloud-functions-framework": "^0.3"
+        "google/cloud-functions-framework": "dev-master as 0.6.0"
```

Comment:
Is this downloading the latest code available directly from GitHub? (i.e. disabling version pinning)

(This is a beta version, so I'm willing to allow it for now - but we **definitely** shouldn't do this once we release publicly.)

Reply:
it's only for testing. I will change it back once a version is tagged